symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py

```diff
@@ -0,0 +1,47 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    angle_type,
+    dimensionless,
+    validate_input,
+    validate_output,
+    print_expression,
+)
+
+# Description
+## For any (positive) value of the damping ratio, the position of a damped oscillator
+## is subject to exponential decay, i.e. its position is proportional to exp(-lambda*t)
+## where lambda is the exponential decay constant and t is time.
+
+# Law: lambda = omega * zeta
+## lambda - exponential decay constant
```

Comment:
It is not really a constant. Maybe `decay rate` is better?

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/f2097325-0eaf-4b08-a3d1-40ff1537f952)
It's not a constant, but the terminology seems to be like that

Reply:
ok